Common/Basic components/CartTable.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using OpenQA.Selenium;
+
+namespace Infrastructure
+{
+    public class CartTable : ComponentBase
+    {
+        public List<IWebElement> Garbage { get; set; }
+        public List<IWebElement> Price { get; set; }
+        public List<IWebElement> NumOfItems { get; set; }
+        private IWebElement Quantity { get; set; }
+
+
+        public CartTable(IWebDriver driver, IWebElement parentElement) : base(driver, parentElement)
+        {
+            try
+            {
+                Garbage = parentElement.FindElements(By.CssSelector(".icon-trash")).ToList();
+                Price = parentElement.FindElements(By.CssSelector(".cart_total .price")).ToList();
+                NumOfItems = parentElement.FindElements(By.CssSelector(".cart_quantity.text-center")).ToList();
+                Quantity = base.parentElement.FindElement(By.CssSelector("#summary_products_quantity"));
+            }
+
+            catch
+            {
+                Quantity = base.parentElement.FindElement(By.CssSelector(".alert.alert-warning"));
+            }
```

Comment:
If you don't have any items in your cart, why catch the error? I'm not sure I would find each element in the constructor, rather I would find them normally. If an exception is thrown for not finding the price for instance, it's a problem just like not finding the quantity, it means something is wrong with the site.. 

If you want to check an error appears on purpose, you can make a test for it where you expect there to be an error and then you can find that element